platform/javascript/detect.py

```diff
@@ -69,9 +69,9 @@ def configure(env):
             exec(f.read(), em_config)
         except StandardError as e:
             raise RuntimeError("Emscripten configuration file '%s' is invalid:\n%s" % (em_config_file, e))
-    if 'EMSCRIPTEN_ROOT' not in em_config:
-        raise RuntimeError("'EMSCRIPTEN_ROOT' missing in Emscripten configuration file '%s'" % em_config_file)
-    env.PrependENVPath('PATH', em_config['EMSCRIPTEN_ROOT'])
+    if 'BINARYEN_ROOT' not in em_config and 'EMSCRIPTEN_ROOT' not in em_config:
+        raise RuntimeError("'BINARYEN_ROOT' or 'EMSCRIPTEN_ROOT' missing in Emscripten configuration file '%s'" % em_config_file)
+    env.PrependENVPath('PATH', em_config['BINARYEN_ROOT'] or em_config['EMSCRIPTEN_ROOT'])
```

Comment:
Are you sure this `or` works as you expect in Python? Don't this prepend a bool to the PATH?

Reply:
It seems like I forgot about KeyErrors in Python :no_mouth:

Fixed it by using `get()` with a default value if the key isn't found.